test/unit/request-body.test.js

```diff
@@ -525,14 +526,24 @@ describe('RequestBody', function () {
             expect(rBody.toJSON()).to.eql(definition);
         });
 
-        it('should not have content in file body', function () {
+        it('should have content in file body', function () {
             var definition = {
                     mode: 'file',
-                    file: { src: 'fileSrc', content: 'this should be removed' }
+                    file: { src: 'fileSrc', content: 'this should not be removed' }
                 },
                 rBody = new RequestBody(definition);
 
-            expect(rBody.toJSON().file).to.not.have.property('content');
+            expect(rBody.toJSON().file).to.have.property('content');
+        });
+
+        it('should correctly handle request body with non serializable data', function () {
+            var definition = {
+                    mode: 'file',
+                    file: { content: new Stream.Readable() }
+                },
+                rBody = new RequestBody(definition);
+
+            expect(rBody.toJSON()).to.eql(definition);
```

Comment:
What would the result of serialisation of a `ReadStream` be? Wouldn't it be unusable? Do we want to keep this? @codenirvana

Reply:
`toJSON()` will have no effect on the `readStream`. It will remain usable, I have checked this.
Regarding wether we want to keep it @codenirvana can share his opinion.